Delegates/CardVisibilityDelegate.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  CardVisibilityDelegate.swift
+//  CardParts
+//
+//  Created by Tumer, Deniz on 5/23/18.
+//
+
+/*
+ * This protocol handles passing data about the visibility of a card
+ * from an instance of a CardsViewController to the card itself
+ */
+@objc public protocol CardVisibilityDelegate {
+    
+    /*
+     * Notifies a card of its visibility.
+     *
+     * @param percentVisible - percentage of the card that is visible in the list of cards
+     */
+    @objc optional func cardVisibility(percentVisible: CGFloat)
```

Comment:
Should we include a boolean `isVisible` as well? I could imagine users would want a quick check if it is visible or not and then some might care about the percent visible. Instead of having to check `if percentVisible == 0.0` What do you think?

Reply:
The only thing was that this function doesn't get called unless the card is visible so I thought it was unnecessary. But maybe we could include in the documentation that this method will only be called when the card is actually visible in some way